### DISCUSSION TITLE
Make ESLint + Prettier optional

### DISCUSSION THREAD

**Discussion by angel-luis** [vercel/turborepo#8098] [Ideas]
### Goals

1. We're forced to use ESLint + Prettier when creating a new monorepo
2. ESLint and Prettier are slow for big monorepos

### Background

I use Biomejs because is based in Rust and is much faster for linting and formatting the code that ESLint and Prettier. This speed is specially needed for monorepos, where are much more files than in a standard repo.

### Proposal

When using the command to create the monorepo, add a CLI question to opt-out to ESLint + Prettier.

**Comment by tyleralbee** [comment]
How has the switch been for you - would you suggest switching from both Prettier and ESLint in favor of Biome? It looks very cool.

Are there equivalents to plugins such as `eslint-config-turbo` when using Biome? What kind of configuration are you using for your linter with respect to Turborepo? Are you using a shared configuration package (e.g. `eslint-config` in most of the Turborepo examples)?

**Reply by angel-luis** [reply]
> How has the switch been for you - would you suggest switching from both Prettier and ESLint in favor of Biome? It looks very cool.
> 
> Are there equivalents to plugins such as `eslint-config-turbo` when using Biome? What kind of configuration are you using for your linter with respect to Turborepo? Are you using a shared configuration package (e.g. `eslint-config` in most of the Turborepo examples)?

In my opinion there is still work needed from both Vercel and Biome team. I wouldn't consider switch to Biome until these two requested are completed: 
- https://github.com/biomejs/biome/issues/2228
- https://github.com/vercel/next.js/discussions/59347

I've opened this request because it's the final step to get working Biome in Turborepo, but still there are problems when using Biome in monorepos, and if you use Next.js, the framework is still attached to Eslint v8.
